apmpackage/apm/0.1.0/manifest.yml

```diff
@@ -35,12 +35,94 @@ policy_templates:
         title: Secret token
         required: false
         show_user: true
+      - name: api_key_enabled
+        type: bool
+        title: Enable API Key Auth from APM Agents
+        required: false
+        show_user: true
+        default: false
       - name: enable_rum
         type: bool
         title: Enable RUM
         required: true
         show_user: true
         default: false
+      - name: api_key_limit
+        type: int
+        title: Maximum number of allowed API Keys
+        required: false
+        show_user: false
+        default: 100
+      - name: capture_personal_data
+        type: bool
+        title: Capture Personal Data (e.g. IP, User Agent)
+        required: false
+        show_user: false
+        default: true
+      - name: rum_allow_origins
+        type: string
+        title: RUM - Allow Origins
+        multi: true
+        required: false
+        show_user: false
+        default: ['*']
+      - name: rum_allow_headers
+        type: string
+        title: RUM - Access-Control-Allow-Headers
+        multi: true
+        required: false
+        show_user: false
+      - name: rum_library_pattern
+        type: string
+        title: RUM - pattern identifying library frames
+        required: false
+        show_user: false
+      - name: rum_exclude_from_grouping
+        type: string
+        title: RUM - pattern excluding frames from grouping
+        required: false
+        show_user: false
```

Comment:
```suggestion
```

I know we discussed adding these, but I'm having second thoughts. Let's add them in once we've added support for sourcemaps?

Reply:
SGTM